NetworkOfPrivateClinics/NetworkOfPrivateClinics/ClinicRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NetworkOfPrivateClinics.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetworkOfPrivateClinics
+{
+    public class ClinicRepository : IClinicRepository
+    {
+        public ClinicRepository(List<Clinic> clinicContext) 
+        {
+            context = clinicContext;    
+        }
+
+        private readonly List<Clinic> context;
```

Comment:
private readonly fields should be above the constructor and with underscore prefix `_context`